Mathlib/GroupTheory/Coset.lean

```diff
@@ -809,6 +809,54 @@ theorem card_comap_dvd_of_injective (K : Subgroup H) (f : α →* H)
 
 end Subgroup
 
+namespace MonoidHom
+
+variable [Group α] {H : Type*} [Group H]
+
+/-- An equivalence between any fiber of a `MonoidHom` and its kernel. -/
```

Comment:
```suggestion
/-- An equivalence between any non-empty fiber of a `MonoidHom` and its kernel. -/
```
(I think the empty sets sitting over points not in the range count as fibers too?)

Reply:
This issue also occurs below.